canonical/juju-dashboard
# WD-3995 - Display Juju logo when not using JAAS

## Done

- Display correct login logo when not using JAAS.

## QA

- Connect your dashboard to JIMM and go to the login screen, you should see the JAAS logo.
- Connect your dashboard to a Juju controller and go to the login screen, you should see the Juju logo.

## Details

- https://warthogs.atlassian.net/browse/WD-3995

## Screenshots

<img width="349" alt="Screenshot 2023-05-22 at 2 28 57 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/ab6d7687-e959-4c2c-96d0-3578487b50ae">
<img width="380" alt="Screenshot 2023-05-22 at 2 29 41 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/7748ea7f-e1f1-4d16-9312-cbcc7f2dcf70">
